packages/cef/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.0.0"
+  changes:
+    - description: Update to ECS 8.2
```

As a user I would want more details.

```suggestion
    - description: Update to ECS 8.2 by modifying Check Point events to use the new email field set.
```